.changeset/pre.json

```
@@ -2,7 +2,7 @@
   "mode": "pre",
   "tag": "alpha",
   "initialVersions": {
-    "@mastra/client-js": "0.1.9",
+    "@mastra/client": "0.1.9",
```

let's start at 0.2.0
```suggestion
    "@mastra/client": "0.2.0",
```